pubspec.yaml

```diff
@@ -44,6 +45,9 @@ dependencies:
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
+  fimber: ^0.6.6
+  get_it: ^7.2.0
+  package_info: ^2.0.0
```

Comment:
It seems like this plugin doesn't work on linux and is deprecated, [package_info_plus](https://pub.dev/packages/package_info_plus) works though, but would require a dependency update in dart_wormhole_william

Reply:
Thank you for that finding. Going for `package_info_plus` was prevented from https://github.com/LeastAuthority/dart_wormhole_william using `ffi` in version 1.1.1. Will try to update that.